b2sdk/transfer/inbound/download_manager.py

```diff
@@ -93,8 +92,7 @@ def download_file_from_url(
         ) as response:
             metadata = FileMetadata.from_response(response)
             if range_ is not None:
-                if 'Content-Range' not in response.headers:
-                    raise UnexpectedCloudBehaviour('Content-Range header was expected')
+                # 2021-05-20: unfortunately for a read of a complete object server does not return the 'Content-Range' header
```

Comment:
well actually, we can determine whether server responded with the whole file or not by http status: 200 = full file and no `Content-Range` header; 206 = part of the file and `Content-Range` present

Reply:
B2Http response does not guarantee `.status_code` and I don't want to add it right now because we are not sure how well it is going to work with curl later, while having or not having this header doesn't really have that much impact. I don't think it's worth to do this well, even if it's possible. Do you think we should put the work in to keep the exception raising working?

Reply:
we discussed it privately and determined it's not worth the effort